website/pages/index.tsx

```diff
@@ -271,7 +267,12 @@ export default function Home({ updates }: { updates: Array<UpdateProps> }) {
             <div className={largeHeight}>
               <MarketingPreview
                 marketingURL="https://www.mongodb.com/blog/post/meet-our-product-design-team-part-1"
+<<<<<<< HEAD
                 backgroundURL={`${landingURL}/team-thumbnail.png`}
+=======
+                backgroundURL="/images/team-thumbnail.png"
+                isTouchDevice={isTouchDevice}
+>>>>>>> main
```

Comment:
```suggestion
                backgroundURL={`${landingURL}/team-thumbnail.png`}
                isTouchDevice={isTouchDevice}
```

Reply:
whoops not sure how that snuck in there.. it's gone now